app/assets/stylesheets/frontend/views/_html-publications.scss

```diff
@@ -286,6 +286,71 @@
           }
         }
       }
+
+      table.financial-data {
+
+        .numeric {
+          text-align: right;
+          font-family: $NTA-Light-Tabular;
+        }
+
+        // make all elements inside thead look the same
+        // (td can sometimes be used to overcome complexity of multiple lines in thead)
+        // and make all of thead and tfoot stand out
+        thead,
+        tfoot {
+          background-color: #e1e8e8;
```

Comment:
It turned out Mia liked `$grey-3` much better anyway. :) I just changed it accordingly.